main.py

```diff
@@ -1,7 +1,51 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    create_genres()
+    create_actors()
+    update()
+    delete()
+    return specific_actors()
+
+
+def create_genres() -> None:
+    for name in ["Western", "Action", "Dramma"]:
+        Genre.objects.create(name=name)
+
+
+def create_actors() -> None:
+    actors = [
+        "George Klooney",
+        "Kianu Reaves",
+        "Scarlett Keegan",
+        "Will Smith",
+        "Jaden Smith",
+        "Scarlett Johansson"
+    ]
+    for actor in actors:
```

Comment:
you can use unpacking here - would be more pythonic and saves you one line of code :)